src/ui/public/registry/doc_table_detail_views.js

```diff
@@ -0,0 +1,7 @@
+define(function (require) {
+  return require('ui/registry/_registry')({
+    name: 'docTableDetailViews',
+    index: ['title'],
+    order: ['order']
```

Comment:
In order to implement the defaults I mentioned above you will either want to create a `DocView` class or add a constructor here that can modify the registry when it is injected into the application. I personally prefer the class approach, but it's up to you.

```js
return require('ui/registry/_registry')({
  name: 'docTableDetailViews',
  constructor() {
    this.forEach(docView => {
      docView.shouldShow = docView.shouldShow || _.constant(true)
    })
  }
});
```